### ISSUE TITLE
Needs to gain functionality for warbanks

### ISSUE THREAD

**Issue by DCCXVIII** [Cidan/BetterBags#499]
### Describe the feature

- Needs a way to distinguish between depositing reagents into warbank vs reagent tab. 

- No way to deposit gold into warbank. 

- Tab names should be renamed to reflect the name warbank.

### Describe the implementation

?

**Comment by Cidan** [comment]
This is actively being worked on -- check the Discord for more information.

**Comment by AMaribo** [comment]
Depositing and withdrawing gold, renaming warbank tabs, and the whole tab editing functionality has been added in v0.1.83.
Still missing is the ability to add icons to tab names.

**Comment by avohaj** [comment]
I just deposited the Memories from the prepatch event into my warbank, they appeared to be in the warbank, but when I logged on another character the items weren't there. When I logged back on the character who deposited, they appeared to be in the warbank and I could withdraw them.

I then deposited the items again, disabled BetterBags and they were not shown in the Warbank. I also  looked through all my regular bank bags, but the items weren't there either, when I enabled BetterBags again I could see the items in the warbank and withdraw them. This doesn't happen all the time because earlier I deposited the same items with another character and had no problems withdrawing them again with another.

I don't know in what kind of void the items landed, but I'm happy that I didn't lose anything, but things are definitely still wonky.

Also, when I had the second warbank tab selected and right clicked items, they still got deposited into the first bank tab, manual drag&drop into the second tab worked.

**Comment by AMaribo** [comment]
The first part of your message might be a bug. Posting it as its own issue would help track it.


> Also, when I had the second warbank tab selected and right clicked items, they still got deposited into the first bank tab, manual drag&drop into the second tab worked.

This portion is how Blizzard has implemented depositing items into the Warbank. If you want the items to go to different tabs, you'll need to set their affinities for those items. Otherwise the items will go into the first available tab.

**Comment by avohaj** [comment]
> The first part of your message might be a bug. Posting it as its own issue would help track it.

Yeah sorry nevermind, I had the same happen the default UI now too. It's actually because they're Unique(20) and I had 15 in my bags and 6 in the warbank - for some reason it then completely hides the item in the warbank if it could cause that character to withdraw an "illegal amount".
